tests/test_tornado.py

```diff
@@ -1,28 +1,34 @@
 import unittest
-from tornado import gen
+import tornado
+from tornado.testing import gen_test
 from tornado.ioloop import IOLoop
 from gremlinclient import submit
 
 
-class Py27SyntaxTest(unittest.TestCase):
+class Py27SyntaxTest(tornado.testing.AsyncTestCase):
 
     def setUp(self):
+        super(Py27SyntaxTest, self).setUp()
         self.loop = IOLoop.current()
 
+    @gen_test
     def test_submit(self):
 
-        @gen.coroutine
-        def go():
-            f = submit("1 + 1")
-            res = yield f
-            while True:
-                msg = yield res.read()
-                if msg is None:
-                    break
-                self.assertEqual(msg.status_code, 200)
-                self.assertEqual(msg.data[0], 2)
-
-        self.loop.run_sync(go)
```

Comment:
Can we maintain the ``while`` syntax here to make sure the client returns ``None`` after it is done reading the server response? In general the API for ``gremlinclient`` will follow this pattern.